src/libraries/Microsoft.PowerFx.Core/Functions/IAsyncTexlFunction4.cs

```diff
@@ -11,6 +11,6 @@ namespace Microsoft.PowerFx.Core.Functions
     // A Texl function capable of async invokes, using TimeZoneInfo and IRContext. 
     internal interface IAsyncTexlFunction4
     {
-        Task<FormulaValue> InvokeAsync(TimeZoneInfo timezoneInfo, FormulaType irContext, FormulaValue[] args, CancellationToken cancellationToken);
+        Task<FormulaValue> InvokeAsync(TimeZoneInfo timezoneInfo, FormulaType irContext, FormulaValue[] args, Action checkCancellation);
```

Comment:
>checkCancellation [](http://example.com/codeflow?start=117&length=17)

Instead of updating this interface, can we move the functions to IAsyncTexlFunction5, so that at least we'll be closer to merging all of those into a single interface to be able to finally remove them?